### PR TITLE
Feature/update makefile and travis

### PR DESCRIPTION
**Pull request by gemmai95**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ python:
     - "3.5"
     - "3.4"
 before_install:
-    - git clone https://github.com/ONSdigital/sdx-common.git
+    - git clone --branch 0.7.0 https://github.com/ONSdigital/sdx-common.git
```
Comment (warrenbailey) [inline .travis.yml]: I'm not keen on the version of sdx-common being in 3 places now. 

We could rework the travis script to at call the make file, that would remove the need to have this line twice at least.

**Comment by gemmai95** [comment]
[(2) Remove erroneous use of SDX_HOME in makefiles](https://trello.com/c/yaTznQmn/190-2-remove-erroneous-use-of-sdxhome-in-makefiles)

File: .travis.yml [hunk 2]
```diff
@@ -2,15 +2,11 @@ language: python
...
 install:
-    - pip install -r requirements.txt
-    - pip install -r test_requirements.txt
+    - make build
     - pip install codecov
 script:
-    - flake8
+    - make test
     - python -m unittest tests/test_service.py
     - coverage run -m unittest tests/*.py
```
Comment (necrophonic) [inline .travis.yml]: The `make test` already runs the unit test step - still require the coverage though

File: .travis.yml (hunk 1 above)
Comment (necrophonic) [inline .travis.yml]: Tech debt card has been raised to address this properly

**Pull request merged**